README.md

```diff
@@ -1,2 +1,10 @@
 # code-genie-server
 server for code genie
+
+# Environment Variables
+
```

Comment:
add explenation that the env vars are saved in bashrc, and how they are loaded in the code.
explain that any user should save the following exact keys